packages/server/src/cli.ts

```diff
@@ -235,6 +235,9 @@ const bold = (text: string | number): string | number => {
 		} : undefined,
 	});
 
+	if (options.noAuth && options.host === "0.0.0.0"|| options.allowHttp && options.host === "0.0.0.0") {
```

Comment:
Add space between between the string and the pipes `0.0.0.0"||`

Reply:
added